### ISSUE TITLE
[Adaptive Cards - Home Page] - 'Visit the hub' control present on the main section of the page visually appears as normal static texts in 'Aquatic and Desert' modes.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9301]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)

**Pre-requisites:**
Turn on 'Aquatic/Desert' theme' (**Navigation Path:** System Settings> Accessibility > Contrast theme>Aquatic/Desert theme)

**Repro Steps:**

1. Open the above URL on the Edge Dev browser.
2. Navigate to the 'Visit the hub' control present on the main section of the page using tab key.
3. Verify that, 'Visit the hub' control present on the main section of the page visually appears as normal static texts in 'Aquatic and Desert' modes or not.

**Actual Result:**
'Visit the hub' control present on the main section of the page visually appears as normal static texts in 'Aquatic and Desert' modes.

**Note:**
Same issue replicates for the 'Visit the hub' links present on the 'Schema Explorer', 'Samples' and 'Blog' page

**Expected Result:**
'Visit the hub' control present on the main section of the page should be properly appear as interactive 'hyperlink' control as per the format of in 'Aquatic and Desert' modes. They must not display as static text in any theme.

**User Experience:**
People with visual disabilities who rely on high contrast themes to see content properly will be impacted if the 'Visit the hub' control appear as plain text in 'Aquatic' and 'Desert' modes. Without clear visual identification or interactive elements, users may be unable to identify the control. This limits their ability to navigate or refine data on the page. As a result, accessibility and usability are significantly reduced for affected users.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/79744dc9-143a-47a2-b5c9-cbf43c3e9edc)

### Card JSON

```JSON
NA
```
